pkg/report/template.go

```diff
@@ -0,0 +1,109 @@
+package report
+
+import (
+	"reflect"
+	"strings"
+	"text/template"
+
+	"github.com/containers/common/pkg/report/camelcase"
+)
+
+// Template embeds template.Template to add functionality to methods
+type Template struct {
+	*template.Template
+	isTable bool
+}
+
+// FuncMap is aliased from template.FuncMap
+type FuncMap template.FuncMap
+
+// tableReplacer will remove 'table ' prefix and clean up tabs
+var tableReplacer = strings.NewReplacer(
+	"table ", "",
+	`\t`, "\t",
+	`\n`, "\n",
+	" ", "\t",
+)
+
+// escapedReplacer will clean up escaped characters from CLI
+var escapedReplacer = strings.NewReplacer(
+	`\t`, "\t",
+	`\n`, "\n",
+)
+
+// NormalizeFormat reads given go template format provided by CLI and munges it into what we need
+func NormalizeFormat(format string) string {
+	var f string
+	// two replacers used so we only remove the prefix keyword `table`
+	if strings.HasPrefix(format, "table ") {
+		f = tableReplacer.Replace(format)
```

Comment:
We already talked about this but leaving a comment here for consistency: 
`--format "table {{.Field}}"`  =  headers
`--format "{{.Field}}"`  = no headers

Reply:
I think a more specific comment about this choice here about it here would make sense